extensions/wikia/FacebookClient/FacebookMapModel.class.php

```diff
@@ -29,6 +29,12 @@ class FacebookMapModel {
 	const ERROR_FACEBOOK_USER_ID_MISMATCH = 1;
 	const ERROR_WIKIA_USER_ID_MISMATCH = 2;
 
+	// This is the App ID given to records created before we started collecting the App ID
+	const APP_ID_DEFAULT_V1 = 0;
```

Comment:
Since we have this handy constant now, can we swap out where were just using "0" before? [Here](https://github.com/Wikia/app/blob/dev/extensions/wikia/FacebookClient/FacebookMapModel.class.php#L192), and [here](https://github.com/Wikia/app/blob/dev/extensions/wikia/FacebookClient/FacebookMapModel.class.php#L384).